packages/pixeloven/cli/src/templates/project/package.json.ejs

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "<%= props.projectName %>",
+    "author": "<%= props.projectAuthor %>",
+    "description": "<%= props.projectDescription %>",
+    "private": true,
+    "workspaces": [
+      "apps/*",
+      "packages/*"
+    ],
+    "devDependencies": {
+      "@pixeloven/cli": "6.0.0-rc.0",
```

Comment:
Is there any way to get these dynamically?

Reply:
Yeah I have a plan for that.